.agents/specs/design/HANDOFF-TERMS.md

```diff
@@ -0,0 +1,201 @@
+# Handoff Terms Specification
+
+**Purpose**: Define consistent terminology for agent handoffs, verdicts, and status indicators across all workflows.
+
+**Scope**: This document is the single source of truth for handoff vocabulary between agents, GitHub workflows, and orchestration systems.
+
+## Quick Reference
+
+| Category | Terms | Used By |
```

Comment:
@rjmurillo-bot perform CVA on these terms. Seems like we can reconcile to a single, unified domain language to _consistently_ handle handoffs and handoff logic. We should then have an ADR to codify that.